### PR TITLE
Protect onRefreshGraphics while hot reloading

### PR DESCRIPTION
**Pull request by jasonLaster**
small hot-reloading fix

### CODE REVIEW

**Comment by jaril** [comment]
This gets rid of the error, but it ends up getting stuck in a blank loading screen.

Demo for forcing fast refresh:
https://app.replay.io/recording/bfb175b9-5eb0-4bbe-abfa-8e5141df59cf

Not sure about this yet, but there's a larger problem which is the way that we bootstrap the app is not [fast refresh friendly](https://nextjs.org/docs/basic-features/fast-refresh).

File: src/protocol/graphics.ts [hunk 1]
```diff
@@ -515,10 +515,9 @@ export function refreshGraphics() {
       cx.drawImage(image, 0, 0);
     }
 
-    onRefreshGraphics({
-      ...bounds,
-      gDevicePixelRatio,
-    });
+    if (onRefreshGraphics) {
+      onRefreshGraphics({ ...bounds, gDevicePixelRatio });
```
Comment (jcmorrow) [inline src/protocol/graphics.ts:519]: Is this a place for `onRefreshGraphics?.({ ...bounds, gDevicePixelRatio });`?

**Review by jcmorrow — APPROVED** [review]
Happy with this fix! 👍

**Comment by jcmorrow** [comment]
> Happy with this fix! 👍

Lol, nvm I did not see Jaril's message!

File: src/protocol/graphics.ts (hunk 1 above)
Comment (jasonLaster) [reply src/protocol/graphics.ts:519]: yep!

**Comment by jasonLaster** [comment]
closing since this is a bigger bug

**Pull request closed without merging**